egs/mini_an4/asr1/cmd.sh

```diff
@@ -28,7 +28,7 @@
 
 
 # Select the backend used by run.sh from "local", "stdout", "sge", "slurm", or "ssh"
-cmd_backend='local'
+cmd_backend='stdout'
```

Comment:
I remembered `stdout` does not stop when the error happened.
Is it OK?

Reply:
I found it was not true. Now stdout.pl correctly stopped when the exist status != 0. See  https://github.com/espnet/espnet/runs/1105039198#step:9:46

I suspect that maybe we forgot `set -euo pipefail`

Reply:
Well, however, I'm not sure. I will revert this.